tests/unit/pages/subjects/Subjects.spec.jsx

```diff
@@ -0,0 +1,31 @@
+import { render, screen } from '@testing-library/react'
+import '@testing-library/jest-dom'
+import Subjects from '~/pages/subjects/Subjects'
+import { ModalProvider } from '~/context/modal-context'
```

Comment:
I do not like using a real provider here, but can agree with it for speeding up our developing process.